src/js/view/utils/views-manager.js

```diff
@@ -78,19 +85,29 @@ if (isAndroidChrome && hasOrientation) {
 window.addEventListener('beforeunload', () => {
     document.removeEventListener('visibilitychange', onVisibilityChange);
     document.removeEventListener('webkitvisibilitychange', onVisibilityChange);
+    document.removeEventListener('webkitvisibilitychange', onVisibilityChange);
+    window.removeEventListener('scroll', onScroll);
 
     if (isAndroidChrome && hasOrientation) {
         window.screen.orientation.removeEventListener('change', onOrientationChange);
     }
 });
 
+window.addEventListener('scroll', onScroll);
```

Comment:
npick but can this line go up to ln 80 to be grouped with other `addEventListener` calls?

Reply:
Maybe we don't need to do this until a scroll handler is added?

Reply:
great calls